src/test/java/org/cbioportal/cdd/TestClinicalDataDictionaryApplication.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2019 Memorial Sloan-Kettering Cancer Center.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY, WITHOUT EVEN THE IMPLIED WARRANTY OF MERCHANTABILITY OR FITNESS
+ * FOR A PARTICULAR PURPOSE. The software and documentation provided hereunder
+ * is on an "as is" basis, and Memorial Sloan-Kettering Cancer Center has no
+ * obligations to provide maintenance, support, updates, enhancements or
+ * modifications. In no event shall Memorial Sloan-Kettering Cancer Center be
+ * liable to any party for direct, indirect, special, incidental or
+ * consequential damages, including lost profits, arising out of the use of this
+ * software and its documentation, even if Memorial Sloan-Kettering Cancer
+ * Center has been advised of the possibility of such damage.
+ */
+
+package org.cbioportal.cdd;
+
+import org.cbioportal.cdd.config.CDDAppConfig;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.ContextConfiguration;
+//import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+/**
+ *
+ * @author ochoaa
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes={CDDAppConfig.class})
+//@TestPropertySource("classpath:application.properties")
+public class TestClinicalDataDictionaryApplication {
+    @Test
+    public void testDefault() {
+        System.out.println("PLACEHOLDER TEST");
```

Comment:
Maybe this whole class is not necessary?
If we keep this class, we should follow the same naming scheme for test classes .. either:
{TestedClassName}Test  or Test{TestedClassName}